### DISCUSSION TITLE
WolfHud mod (from Payday 2) is crashing during the loading.

### DISCUSSION THREAD

**Discussion by Xandinhoacqua** [community/community#58485] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I've downloaded wolfhud mod today, but i cant use it bcs crashes my game. This is what says in the crash report: "mods/WolfHUD-master/lua/CustomHUD.lua:4143: attempt to call method 'update' (a nil value)". Does anyone knows how to fix it?

**Comment by airtower-luna** [comment]
It looks like you're looking for help with a software hosted on GitHub, maybe https://github.com/Kamikaze94/WolfHUD? In that case you should look for support information *in that repository*, usually in the README or a file called SUPPORT. If those don't say anything, opening an issue there is usually good. :slightly_smiling_face:
